File: javascript/raindrops/raindrops.js
export const convert = number => {
  let string = "";
  if (number % 3 == 0) string += "Pling";
  if (number % 5 == 0) string += "Plang";
  if (number % 7 == 0) string += "Plong";
  if (!string) return String(number);
  return string;
};

File: python/raindrops/raindrops.py
RULES = (
    (3, "Pling"),
    (5, "Plang"),
    (7, "Plong")
)

def convert(number):
    return "".join(
        drop
        for factor, drop in RULES
        if number % factor == 0
    ) or str(number)
